spartan/aztec-network/templates/pxe.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.network.enableBots }}
+{{- if not .Values.pxe.external }}
```

Comment:
Is there a big reason for the bot to just run its own pxe?

Reply:
Overall given the nature of the pxe the external setup I find surprising

Reply:
No, it shouldn't be necessary to include the PXE in the bot. It should be supported to pass a PXE url to the bot when starting it. I don't know if that actually works.

Reply:
sorry to NOT just run its own PXE*